### PR TITLE
Add subdomain and redirect for explorations.conversa.g0v.network

### PR DESCRIPTION
**Pull request by patcon**
Uses a tool built by ronnywang of g0v

How's this look @ballPointPenguin @dsiddarth? Adding us as points of contact just bc we were on video, but might want to talk out how to decide who goes there (laaaaater not urgent)

EDIT: "xxxxxxxx" being whatever we eventually choose as our namespace (avoiding using "polis")

EDIT2: "conversa" is the tentative name for any related convos for now, as decided on the [2021-03-13 open call](https://hackmd.io/@patcon/polis-calls/https%3A%2F%2Fhackmd.io%2F%40patcon%2Fr10fNCWQ_)

### CODE REVIEW

**Comment by patcon** [comment]
Also, we could decide to use `explorations.polis.g0v.network`, and then others can have stuff at the `*.polis.g0v.network` namespace 🤷‍♂️

**Comment by patcon** [comment]
cc @devinbalkind, as he was interested in governance of this domain, and maybe the first PR is a good place to notice what's needed :)

File: g0v.network./polis-explorations.g0v.network.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+polis-explorations:
+  - type: txt
```
Comment (patcon) [inline g0v.network./polis-explorations.g0v.network.yml:3]:
```suggestion
  - type: TXT
```

**Comment by ballPointPenguin** [comment]
Do you think it would be better to link to the Playlist Overview page?
https://www.youtube.com/playlist?list=PLSL_F7Lwul2dlI9J6-Nzz9_9Pz9AbcG2z

File: g0v.network./polis-explorations.g0v.network.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+polis-explorations:
+  - type: TXT
+    value: 301 https://www.youtube.com/watch?v=pVJE6GXqzsw&list=PLSL_F7Lwul2dlI9J6-Nzz9_9Pz9AbcG2z
```
Comment (patcon) [inline g0v.network./polis-explorations.g0v.network.yml:4]:
```suggestion
    value: 301 https://www.youtube.com/playlist?list=PLSL_F7Lwul2dlI9J6-Nzz9_9Pz9AbcG2z
```

good call @ballPointPenguin

**Review by ballPointPenguin — APPROVED** [review]

**Comment by patcon** [comment]
Thanks Bennie! While we get going, not going to rush merging this, which feels like a safe way to ease into a shared governance process :)

Will commit to waiting till after open call (Sat, Feb 20), so Divya and Devin and anyone else to weigh in on how we do this

File: g0v.network./polis-explorations.g0v.network.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+polis-explorations:
```
Comment (patcon) [inline g0v.network./polis-explorations.g0v.network.yml:2]: If we want to redirect `explorations.polis.g0v.network` instead:

```suggestion
explorations:
```

Would also need to rename file
`g0v.network./polis-explorations.g0v.network.yml` => `g0v.network./explorations.polis.g0v.network.yml`

**Comment by patcon** [comment]
options for non-trademark domains:
- explorations.***.g0v.network
  - polls -1 pc
  - p0lis -1 pc
  - conversa? +1 pc, br
    - inspo: https://github.com/conversa-app/conversa.cc
  - afsheen: poli*/poly*

afsheen: cosmopolitian, but getting away from colonial aspects. greek? persians? word for chess. a game. tacta = bed/grave. pajama, maybe with accent.

File: g0v.network./explorations.xxxxxxxx.g0v.network.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+explorations.xxxxxxxx:
+  - type: TXT
+    value: 301 https://www.youtube.com/watch?v=pVJE6GXqzsw&list=PLSL_F7Lwul2dlI9J6-Nzz9_9Pz9AbcG2z
```
Comment (patcon) [inline g0v.network./explorations.xxxxxxxx.g0v.network.yml:4]:
```suggestion
    value: 301 https://www.youtube.com/playlist?list=PLSL_F7Lwul2dlI9J6-Nzz9_9Pz9AbcG2z
```

File: g0v.network./explorations.xxxxxxxx.g0v.network.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+        - patcon
+  - type: ALIAS
+    value: 301.ronny.tw.
+    octodns:
+      cloudflare:
+        proxied: true
+    metdata:
+      repository: https://github.com/ronnywang/301-service
+      maintainer:
+        - ronnywang
```
Comment (patcon) [inline g0v.network./explorations.xxxxxxxx.g0v.network.yml:18]:
```suggestion
```

File: g0v.network./explorations.xxxxxxxx.g0v.network.yml [hunk 6]
```diff
@@ -0,0 +1,11 @@
+---
+explorations.xxxxxxxx:
+  - type: TXT
+    value: 301 https://www.youtube.com/playlist?list=PLSL_F7Lwul2dlI9J6-Nzz9_9Pz9AbcG2z
+    metdata:
+      maintainer:
+        - ballpointpenguin
+        - dsiddarth
+        - patcon
+  - type: ALIAS
```
Comment (patcon) [inline g0v.network./explorations.xxxxxxxx.g0v.network.yml:10]:
```suggestion
  - type: CNAME
```

**Comment by patcon** [comment]
"conversa" is the tentative name for any related convos for now, as decided on the [2021-03-13 open call](https://hackmd.io/@patcon/polis-calls/https%3A%2F%2Fhackmd.io%2F%40patcon%2Fr10fNCWQ_)

**Comment by patcon** [comment]
We just needed something to get to renaming stuff away from "polis", but we agreed to have a check-in in Apr to vote to lock it in, or choose another name completely. Other proposals still welcome

**Comment by patcon** [comment]
k, got the automation working again, and the change plan is here: https://github.com/g0v-network/domains/pull/7#issuecomment-787638218

Since we decide on this tentative name in the call, going to merge this and we'll have explorations.conversa.g0v.network linking to this playlist! A little long, but that's ok :) cc @ThenWho @dsiddarth @ballPointPenguin

**Comment by ThenWho** [comment]
> We just needed something to get to renaming stuff and avoiding the "polis" trademark, but we agreed to have a check-in at end of March to vote to either (1) lock it in, or (2) choose another name completely. Other proposals still welcome!

May I propose sth that means 'together' or similar?

I tried English, Greek, German and Swedish but the good ones are taken or don't work (swedish 'ihop' is good but trademarked, also 'allt' but wrong grammatically. Greek 'mazi' works but offers no intuition whatsoever 🙂).

Maybe someone else would have better luck in the same theme.

**Pull request merged**